src/wallet/wallet_rpc_server.cpp

```diff
@@ -57,6 +57,8 @@ using namespace epee;
 #undef ARQMA_DEFAULT_LOG_CATEGORY
 #define ARQMA_DEFAULT_LOG_CATEGORY "wallet.rpc"
 
```

Comment:
I would increase refresh period up to 60 seconds. 20 seconds might be too often and generates high net usage.

Reply:
It  could be changed with a RPC request.